### ISSUE TITLE
[Feature Request] Split "Clean up downloaded chapter" under Advanced into two options

### ISSUE THREAD

**Issue by GreatestFool** [Jays2Kings/tachiyomiJ2K#159]
### Why/User Benefit/User Problem
As it is now, it functions like it is supposed to I guess? I don't know, there's no notification with a progress bar so no easy way to tell. That might be something to consider, but not really what I'm asking for here.

The option deletes non-existent, partially downloaded and read chapter folders. I really do like cleaning up those non-existent or partially downloaded chapters every once in a while, because I've noticed it tends to forget to delete those when, for example on Mangadex, a scanlator group deletes an older chapter and replaces it with a new one.

### What/Requirements
However, and while I don't know about others, I primarily download to *keep* the manga. Not just to temporarily binge it, so it would be preferable if I could clean up my downloaded chapters but simultaneously *not* also delete all my read chapters at the same time. Having to manually mark each downloaded manga I want to keep as unread would be much too tedious.

**Comment by Nydid** [comment]
I also agree with this, it solved a Bug but creates a Problem for those that like to keep the manga downloaded on their device.
